draft-ietf-rtcweb-jsep.xml

```diff
@@ -2196,11 +2197,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         specified in <xref target="RFC5245"></xref>,
                         Section 15.4.</t>
 
-                        <t>An "a=fingerprint" line for each of the
-                        endpoint's certificates, as specified in <xref
-                        target="RFC4572"></xref>, Section 5; the digest
-                        algorithm used for the fingerprint MUST match
-                        that used in the certificate signature.</t>
+                        <t>One or more "a=fingerprint" line(s) for
+                        each of the endpoint's certificates, as
+                        specified in <xref
```

Comment:
I think you should include some text about the hash function to make the rationale for multi fingerprints more clear, e.g.

For each of the endpoint's certificates, and for each supported digest algorithm, an "a=fingerprint" line computed for the specific certificate and digest algorithm. At least...